cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -953,3 +1050,7 @@ func isSecret(group, version, kind string) bool {
 func isCronJob(group, version, kind string) bool {
 	return group == "batch" && version == "v1" && kind == "CronJob"
 }
+
+func isNetworkPolicy(group, version, kind string) bool {
+	return group == "networking" && version == "v1" && kind == "NetworkPolicy"
```

Comment:
Should the group be `networking.k8s.io`?

Reply:
Yes, it should.
Need to check again.

Reply:
Changed the group name with "networking.k8s.io"